docs/docs/reference/rendering-options/deferred-static-generation.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Deferred Static Generation API
+---
+
+> **Note:** this feature requires running NodeJS server.
+> It is currently fully supported with [`gatsby serve`](/docs/reference/gatsby-cli/#serve) and in [Gatsby Cloud](/products/cloud/).
+
+Deferred Static Generation (DSG) allows you to defer non-critical page generation to the first user request, speeding up build times.
+Instead of generating _every_ page up-front you can decide to build certain pages up-front and others only when a user accesses the page for the first time.
+Subsequent page requests use the same HTML and JSON generated during the very first request to this page.
```

Comment:
```suggestion
Instead of generating _every_ page up front, you can decide to generate certain pages at build time and others only when a user accesses the page for the first time.
Subsequent page requests use the same HTML and JSON generated during the very first request to this page.
```